javascript/index.js

```diff
@@ -31,35 +34,66 @@ function printMilliseconds() {
 }
 
 function printSplit() {
-  // ... your code goes here
+  btnRight.addEventListener('click', () => {
+    if (btnRight.classList.contains('split')){
+    splits.innerHTML = splitValue
+    }
+  })
 }
 
 function clearSplits() {
   // ... your code goes here
 }
 
 function setStopBtn() {
-  // ... your code goes here
+  btnLeft.classList.replace("start", "stop")
+  btnLeft.innerHTML = "STOP"
 }
 
 function setSplitBtn() {
-  // ... your code goes here
+  btnRight.classList.replace("reset", "split")
+  btnRight.innerHTML = "SPLIT"
 }
 
 function setStartBtn() {
-  // ... your code goes here
+  btnLeft.classList.replace("stop", "start")
+  btnLeft.innerHTML = "START"
 }
 
 function setResetBtn() {
-  // ... your code goes here
+  btnRight.classList.replace("split", "reset")
+  btnRight.innerHTML = "RESET"
 }
 
 // Start/Stop Button
 btnLeft.addEventListener('click', () => {
-  // ... your code goes here
+  if (btnLeft.classList.contains('start')){
+  setStopBtn()
+  setSplitBtn()
+  chronometer.startClick()
```

Comment:
The concept of callback and it's implementation was really hard to understand and apply, but it is essential for the code to work. Basically you have a function that updates the DOM (printTime) and you want that to be executed every second, so you want it to run inside the interval. So by passing the function as a parameter here and ten invoke the parameter inside the interval you will be able to achieve that.